app/view_models/nsm/v1/work_item.rb

```diff
@@ -136,6 +136,14 @@ def changed?
         adjustment_comment.present?
       end
 
+      def backlink_path(claim)
+        if any_adjustments?
+          Rails.application.routes.url_helpers.adjusted_nsm_claim_work_items_path(claim, anchor: id)
```

Comment:
The issue is that adjusted work items are visible as line items in both the 'Review and adjust' tab _and_ the 'Adjustments' tab, so it's incorrect to assume that the previous page for the edit screen of an adjusted work item is the 'Adjustments' tab.

Reply:
These are different pages now though, and I imagine it'd depend what you were doing last.

Should be one for UX to decide probably, I can see the case for both